components/steering_wheel/SConscript

```diff
@@ -140,7 +140,8 @@ project_source_files = [
     PROJECT_SOURCE_DIR.File("SystemManager.c"),
     PROJECT_SOURCE_DIR.File("Module.c"),
     PROJECT_SOURCE_DIR.File("CAN/CANIO-tx.c"),
-    PROJECT_SOURCE_DIR.File("IO.c"),
+    PROJECT_SOURCE_DIR.File("IO/IO_Analog.c"),
```

Comment:
we can do this in a future PR, but I think we should actually go with `ADC` for the analog stuff and just `IO` for the digital side.

Reply:
So change the IO_Analog over to ADC.c? Just to confirm proper understanding

Reply:
it's ok you don't have to do it here, we can do that another time. Yeah the filenames as well as struct names and stuff